share/get_shares.go

```diff
@@ -1,16 +1,16 @@
-package ipld
+package share
 
 import (
 	"context"
 	"sync"
 
+	"github.com/celestiaorg/celestia-node/share/ipld"
```

Comment:
Local imports should come after dependencies. Goland can be configured to do this automatically

Reply:
Pls check in other files as well

Reply:
Thank you. I was using MakeFile and turned out there was a mistake that made -local flag to not work. Fixed